plugins/inputs/opcua_listener/README.md

```diff
@@ -0,0 +1,241 @@
+# OPC UA Client Listener Input Plugin
+
+The `opcua_listener` plugin subscribes to data from OPC UA Server devices.
+
+Telegraf minimum version: Telegraf 1.24
+Plugin minimum tested version: 1.24
+
+## Configuration
+
+```toml @sample.conf
+# Retrieve data from OPCUA devices
+[[inputs.opcua]]
```

Comment:
`opcua_listener` a different plugin than opcua? Should all these docs be updated with `opcua_listener`?

Reply:
Yes, copy paste error